maquetado/books-list.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Books list</title>
+  <meta name="description" content="books-list">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <div class="big-container">
+    <div class="books-list-container">
+      <div class="book-preview">
+        <img src ="./assets/book-cover.png" class="book-small-preview">
+        <h1 class="book-preview-info bold">Titulo del libro</h1>
+        <h1 class="book-preview-info detail-color">Autor del libro</h1>
```

Comment:
I would change this html tag for `h2`

Reply:
Only one h1 in the document at the same time: https://www.hobo-web.co.uk/headers/